commit-message.YAML-tmLanguage

```diff
@@ -54,18 +54,26 @@ patterns:
     - name: comment.line.number-sign.git-commit
       match: (^#)\s*(modified:.*)$\n?
       captures:
-        '0': {name: markup.changed.git-commit}
         '1': {name: punctuation.definition.comment.git-commit}
+        '2': {name: markup.changed.git-commit}
+    - name: comment.line.number-sign.git-commit
+      match: (^#)\s*(renamed:)\s*(.*)\s*(->)\s*(.*)$\n?
```

Comment:
`(^#)\s*(renamed:)\s*(.*(?= )) (->)\s*(.*)$` looks OK. And other lines can be left as is IMHO.
But decide yourself. On my color theme it will all look the same anyway.